includes/Core/Util/Activation.php

```diff
@@ -155,7 +155,9 @@ private function get_activation_notice() {
 					ob_start();
 					?>
 					<script type="text/javascript">
-					sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
+					if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
```

Comment:
Including this additional check to prevent console errors in the case user deactivates plugin without going through reset, when reactivating sendAnalyticsTrackingEvent is undefined as we are only printing script when user isn't authenticated.